UtilityFunctions/LukeMeilerHW7P1.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>your sample spacing should be 1 (I think you forget leap years). But that is not correct in units of years (I think that is what you wanted) So you should have a sample spacing what 1 hour is in 33 years.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_GreenhouseGases/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>